README.md

```diff
@@ -10,6 +10,50 @@ intended for testing new features, and can deploy sandboxes automatically from
 GitHub pull requests.
 
 
+What does OpenCraft Instance Manager do and how to use it
+---------------------------------------------------------
+
+This section focuses on how to use the web interface, as opposed to how to install the code,
+debug it or develop it (see next sections for that).
+
+Ocim is a web interface to manage a list of Open edX servers. With some clicks you can create
+new servers, check their status and their configuration.
+It looks like this:
+
+![Ocim's main screen](documentation/ocim_main_screen.png)
```

Comment:
Please make sure the screenshot doesn't contain any client names - this one does.

Reply:
@bradenmacdonald ok, I edited it, the IPs are also fake now. The URL too.
I changed the document too and deleted your comment, which showed the URL in a diff.

Reply:
Thanks @clemente !